package.json

```diff
@@ -75,6 +75,7 @@
     "@types/uuid": "^3.4.5",
     "@typescript-eslint/eslint-plugin": "^2.0.0",
     "@typescript-eslint/parser": "^2.0.0",
+    "@wojtekmaj/enzyme-adapter-react-17": "^0.8.0",
```

Comment:
So enzyme doesnt really support anything `>react@16`, this seems like the only way to get past dependency issues for now, but we should consider moving to `react-testing-library`.

https://www.npmjs.com/package/@wojtekmaj/enzyme-adapter-react-17